private-click-measurement.bs

```diff
@@ -241,6 +241,18 @@ In the interest of user privacy, user agents are encouraged to deploy the follow
 * The user agent offers users a way to turn Private Click Measurement on and off.
 * The user agent doesn't support Private Click Measurement in private/incognito mode.
 
+## Event-Level Measurement Combined With Pre-Existing Tracking Vectors ## {#pre-existing-tracking-vectors}
+
+Any form of event-level measurement, i.e. where there's one report per event, can be combined with pre-existing tracking vectors in an attempt to track users cross-site.
+
+### Combined With Fingerprinting ### {#combined-with-fingerprinting}
+
+One pre-existing tracking vector is device fingerprinting. The click source and click destination may be able to record fingerprints for all measured events based on e.g. IP address, user agent string, and user-installed fonts. These fingerprints would reduce the number of users who would need a unique ID in event-level attribution reports to be uniquelly identified. Combining with fingerprinting is not a problem inherent to event-level meaurement such as Private Click Measurement, but rather an existing privacy problem on the web. User agents have to defend against device fingerprinting to prohibit this kind of attack.
+
+### Combined With Navigational Tracking ### {#combined-with-navigational-tracking}
+
+Another pre-existing tracking vector is [navigational tracking](https://privacycg.github.io/nav-tracking-mitigations/#terminology), sometimes referred to as link decoration tracking. The click source can include a user ID or click ID in the destination URL and the click destination can read and store that ID for attribution purposes. Under the assumption of navigational tracking, there is no need for a feature like Private Click Measurement but it is of course possible to combine navigational tracking with event-level measurement to try to track individual users cross-site. This problem is not inherent to event-level meaurement such as Private Click Measurement, but rather an existing privacy problem on the web. User agents have to defend against navigational tracking to prohibit this kind of attack.
```

Comment:
I'm not clear on your interpretation of the definition here.  Your cited definition is:

> ***Navigational tracking*** refers to the general use of one or more [navigations](https://html.spec.whatwg.org/multipage/browsing-the-web.html#browsing-the-web) to identify that a user on one site is the same person as a user on another site.

I don't consider "link decoration tracking" to be an alternative name for navigation tracking.  I consider it a subset: just one technique that can be used.

I do agree that if you have reliable navigation tracking that uses just a single navigation (not all techniques do), then you don't need PCM.

In that light, I might make this more pointed:

> If sites use link decoration, it is possible that they will be able to use something like link decoration [LINK] to perform attribution.  However, user agents that protect against navigation tracking [LINK] might block the use of link decoration.

Reply:
> I'm not clear on your interpretation of the definition here. Your cited definition is:
> 
> > _**Navigational tracking**_ refers to the general use of one or more [navigations](https://html.spec.whatwg.org/multipage/browsing-the-web.html#browsing-the-web) to identify that a user on one site is the same person as a user on another site.

I include cross-site tracking of _user activity_ across sites and not just _linking of identity_ across sites. Is that your take too? It's not clear from the above.

> I don't consider "link decoration tracking" to be an alternative name for navigation tracking. I consider it a subset: just one technique that can be used.

Fair. I'll update that language.

> I do agree that if you have reliable navigation tracking that uses just a single navigation (not all techniques do), then you don't need PCM.
> 
> In that light, I might make this more pointed:
> 
> > If sites use link decoration, it is possible that they will be able to use something like link decoration [LINK] to perform attribution.  However, user agents that protect against navigation tracking [LINK] might block the use of link decoration.

Would the [LINK] here be to the WHATWG site or the navigational tracking work item?

Reply:
> I include cross-site tracking of user activity across sites and not just linking of identity across sites. Is that your take too? It's not clear from the above.

I agree.  I only included this because we're talking specifically about navigation tracking and this is the definition you cited; so it seemed useful context.

> Would the [LINK] here be to the WHATWG site or the navigational tracking work item?

[LINK] could be whatever you think is good, though the navigation tracking document defines both terms fairly well, so you could use those.